invenio_app_ils/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -18,29 +46,55 @@
           },
           "type": "object"
         },
-        "edition": {
-          "type": "keyword"
+        "abstracts": {
+          "properties": {
+            "source": {
+              "type": "text"
+            },
+            "value": {
+              "type": "text"
+            }
+          },
+          "type": "object"
         },
         "authors": {
           "type": "object",
           "properties": {
+            "affiliations": {
+              "type": "object",
+              "properties": {
+                "curated_relation": {
+                  "type": "boolean"
+                },
+                "value": {
+                  "type": "keyword"
+                }
+              }
+            },
+            "alternative_names": {
+              "type": "keyword"
```

Comment:
There are a lot of "type": "keywords", i.e. `volume`, `note` etc, which I think they should be `text`.